### PR TITLE
Remove coveralls ping from travis

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

**Comment by alexgibson** [comment]
Does this just remove the travis ping, or stop coveralls altogether?

**Comment by schalkneethling** [comment]
From what I recall speaking to @pmclanahan, this will stop coverage from running on the python code and also stop the ping to coveralls. I have been meaning to remove it so we can add the JS unit tests but, if coveralls is misbehaving, perhaps we should just leave it be.

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
 script:
   - grunt test
-  - coverage run manage.py test
+  - manage.py test
```
Comment (pmclanahan) [inline .travis.yml]: has to be `python manage.py test`

**Comment by schalkneethling** [comment]
You know what, I bet that needs to be `./manage.py test

File: .travis.yml (hunk 1 above)
Comment (schalkneethling) [inline .travis.yml]: Thanks! Just figured it needs to be something like that as it is not run by coverage anymore :blush:

**Comment by schalkneethling** [comment]
Whoops, we need to remove the after-success block as well else it still pings coveralls.

**Pull request merged**